core/app/models/spree/stock/package.rb

```diff
@@ -91,6 +91,10 @@ def to_shipment
       def contents_by_weight
         contents.sort { |x, y| y.weight <=> x.weight }
       end
+
+      def volume
+        contents.sum(&:volume)
```

Comment:
It would be best to add a spec for this method so that it isn't accidentally removed.